protocol/messenger_communities.go

```diff
@@ -4581,11 +5316,20 @@ func (m *Messenger) MarkDiscordCommunityImportAsCancelled(communityID string) {
 	m.importingCommunities[communityID] = true
 }
 
+func (m *Messenger) MarkDiscordChannelImportAsCancelled(channelID string) {
+	m.importingChannels[channelID] = true
```

Comment:
We should probably rename this and `importingCommunities` to something like `canceledImportChannels`, because the name is confusing.

Reply:
I'm guessing this will be fixed here: https://github.com/status-im/status-desktop/issues/12504